src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -58,6 +60,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@original-brownbear would you accept such suppression or in this case we should create puzzle to break the class up?